tracks/csharp/exercises/beer-song/beer-song/mentoring.md

```diff
@@ -0,0 +1,44 @@
+## Mentoring
+The student would normally go first using the if/else and if/else if statements.
+Suggesting the use of case statements first will be optimal and instructive. 
+Learning material: [Switch statements ](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/selection-statements)
+
+## The Exemplar Solution
+Try and guide the student towards this solution.
```

Comment:
Rather than just providing a solution and saying, "Make students do this code", can you highlight common pitfalls, suggestions, or directions that help students learn what can be done better? Mentoring should not be showing students code and asking them to memorize/emulate the code. It should be about specific code techniques, pitfalls, etc.

Reply:
Thank you for the instructions, gonna do this!